examples/pytorch/mnist_tensorboard_artifact.py

```diff
@@ -218,3 +220,23 @@ def log_scalar(name, value, step):
         "\nLaunch TensorBoard with:\n\ntensorboard --logdir=%s"
         % os.path.join(mlflow.get_artifact_uri(), "events")
     )
+
+    # Log the model as an artifact of the MLflow run.
+    print("\nLogging the trained model as a run artifact...")
+    mlflow.pytorch.log_model(model, artifact_path="pytorch-model", pickle_module=pickle)
```

Comment:
`pickle_module=pickle` is to suppress the error reported in for https://github.com/mlflow/mlflow/issues/3199?

Reply:
Yes, I faced the exact same issue. Using the work-around you suggested on #3199. Thank you for it!